### DISCUSSION TITLE
I haven't got the benefit of student developer pack after 72hours of the verification? how long does it take? Is says You are verified and your benefit will be available soon.

### DISCUSSION THREAD

**Discussion by mirage763** [community/community#137105] [GitHub Education]
### Hi everyone,

I haven't got the benefit of student developer pack after 72hours of the verification? how long does it take? Is says You are verified and your benefit will be available soon.

**Comment by davevad93** [comment]
Hi @mirage763 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @mirage763 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by Meshariabdullahf** [comment]
Hi everyone,

i need some help to get benefit of student developer pack .. whoever can help me, please contact me. Thank you very much.

my insta : @m7medfaleh
